### ISSUE TITLE
RuntimeError: wrapped C/C++ object of type OWPaintData has been deleted

### ISSUE THREAD

**Issue by DaiZack** [biolab/orange3#4816]
1. used data painter and tree
2. when close orange
3. See error

**Orange version:**
3.25 (install from miniconda)

**Expected behavior**
report error when close canvas

**Screenshots**
![image](https://user-images.githubusercontent.com/45176371/82849563-8462c480-9ec6-11ea-94ec-3aa5c49fa1b4.png)

**Operating system:**
Windows 10

```
C:\Users\dieze\miniconda3\lib\site-packages\orangewidget\widget.py:286: RuntimeWarning: subclassing of widget classes is deprecated and will be disabled in the future.
Extract code from OWBaseVectorizer or explicitly open it.
  RuntimeWarning)

--------------------------- RuntimeError Exception ----------------------------
Traceback (most recent call last):
  File "C:\Users\dieze\miniconda3\lib\site-packages\orangewidget\workflow\widgetsscheme.py", line 512, in eventFilter
    return super().eventFilter(receiver, event)
  File "C:\Users\dieze\miniconda3\lib\site-packages\orangecanvas\scheme\widgetmanager.py", line 600, in eventFilter
    self.__dispatch_events(recv, event)
  File "C:\Users\dieze\miniconda3\lib\site-packages\orangecanvas\scheme\widgetmanager.py", line 619, in __dispatch_events
    QCoreApplication.sendEvent(item.widget, event)
RuntimeError: wrapped C/C++ object of type OWPaintData has been deleted
-------------------------------------------------------------------------------
```

**Comment by ales-erjavec** [comment]
I cannot duplicate this.

Can you please run

```
conda list --explicit
```

and post the output?

**Comment by DaiZack** [comment]
I think it is OK. I found the error only occurs when I quit Orange without saving the project. If I save the project before I quit, no errors.

The mini-conda explicit as below: 

# This file may be used to create an environment using:
# $ conda create --name <env> --file <this file>
# platform: win-64
@EXPLICIT
https://repo.anaconda.com/pkgs/main/win-64/blas-1.0-mkl.conda
https://conda.anaconda.org/conda-forge/win-64/ca-certificates-2020.4.5.1-hecc5488_0.tar.bz2
https://repo.anaconda.com/pkgs/main/win-64/icc_rt-2019.0.0-h0cc432a_1.conda
https://repo.anaconda.com/pkgs/main/win-64/intel-openmp-2020.0-166.conda
https://repo.anaconda.com/pkgs/msys2/win-64/msys2-conda-epoch-20160418-1.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/pandoc-2.9.2.1-0.tar.bz2
https://repo.anaconda.com/pkgs/main/win-64/vs2015_runtime-14.16.27012-hf0eaf9b_1.conda
https://conda.anaconda.org/conda-forge/win-64/winpty-0.4.3-4.tar.bz2
https://repo.anaconda.com/pkgs/msys2/win-64/m2w64-gmp-6.1.0-2.tar.bz2
https://repo.anaconda.com/pkgs/msys2/win-64/m2w64-libwinpthread-git-5.0.0.4634.697f757-2.tar.bz2
https://repo.anaconda.com/pkgs/main/win-64/mkl-2020.0-166.conda
https://repo.anaconda.com/pkgs/main/win-64/vc-14.1-h0510ff6_4.conda
https://conda.anaconda.org/conda-forge/win-64/icu-64.2-he025d50_1.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/jpeg-9c-hfa6e2cd_1001.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/libblas-3.8.0-15_mkl.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/libclang-9.0.1-default_hf44288c_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/libsodium-1.0.17-h2fa13f4_0.tar.bz2
https://repo.anaconda.com/pkgs/msys2/win-64/m2w64-gcc-libs-core-5.3.0-7.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/openssl-1.1.1g-he774522_0.tar.bz2
https://repo.anaconda.com/pkgs/main/win-64/sqlite-3.31.1-he774522_0.conda
https://repo.anaconda.com/pkgs/main/win-64/yaml-0.1.7-hc54c509_2.conda
https://conda.anaconda.org/conda-forge/win-64/zlib-1.2.11-h2fa13f4_1006.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/libcblas-3.8.0-15_mkl.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/liblapack-3.8.0-15_mkl.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/libpng-1.6.37-hfe6a214_1.tar.bz2
https://repo.anaconda.com/pkgs/msys2/win-64/m2w64-gcc-libgfortran-5.3.0-6.tar.bz2
https://repo.anaconda.com/pkgs/main/win-64/python-3.7.6-h60c2a47_2.conda
https://conda.anaconda.org/conda-forge/win-64/zeromq-4.3.2-h6538335_2.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/freetype-2.10.2-hd328e21_0.tar.bz2
https://repo.anaconda.com/pkgs/msys2/win-64/m2w64-gcc-libs-5.3.0-7.tar.bz2
https://repo.anaconda.com/pkgs/main/win-64/pywin32-227-py37he774522_1.conda
https://conda.anaconda.org/conda-forge/win-64/qt-5.12.5-h7ef1ec2_0.tar.bz2
https://repo.anaconda.com/pkgs/main/win-64/menuinst-1.4.16-py37he774522_0.conda
https://repo.anaconda.com/pkgs/main/win-64/asn1crypto-1.3.0-py37_0.conda
https://repo.anaconda.com/pkgs/main/win-64/chardet-3.0.4-py37_1003.conda
https://repo.anaconda.com/pkgs/main/win-64/console_shortcut-0.1.1-4.conda
https://repo.anaconda.com/pkgs/main/win-64/idna-2.8-py37_0.conda
https://repo.anaconda.com/pkgs/main/win-64/powershell_shortcut-0.0.1-3.conda
https://repo.anaconda.com/pkgs/main/win-64/pycosat-0.6.3-py37he774522_0.conda
https://repo.anaconda.com/pkgs/main/win-64/pycparser-2.19-py37_0.conda
https://conda.anaconda.org/conda-forge/win-64/python_abi-3.7-1_cp37m.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/pywinpty-0.5.7-py37_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/pyyaml-5.1.2-py37hfa6e2cd_0.tar.bz2
https://repo.anaconda.com/pkgs/main/win-64/ruamel_yaml-0.15.87-py37he774522_0.conda
https://repo.anaconda.com/pkgs/main/win-64/six-1.14.0-py37_0.conda
https://conda.anaconda.org/conda-forge/win-64/tornado-6.0.4-py37hfa6e2cd_0.tar.bz2
https://repo.anaconda.com/pkgs/main/win-64/win_inet_pton-1.1.0-py37_0.conda
https://repo.anaconda.com/pkgs/main/win-64/wincertstore-0.2-py37_0.conda
https://conda.anaconda.org/conda-forge/win-64/certifi-2020.4.5.1-py37hc8dfbb8_0.tar.bz2
https://repo.anaconda.com/pkgs/main/win-64/cffi-1.14.0-py37h7a1dbc1_0.conda
https://conda.anaconda.org/conda-forge/win-64/docutils-0.16-py37hc8dfbb8_1.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/entrypoints-0.3-py37hc8dfbb8_1001.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/future-0.18.2-py37hc8dfbb8_1.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/kiwisolver-1.2.0-py37heaa310e_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/markupsafe-1.1.1-py37h8055547_1.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/mistune-0.8.4-py37h8055547_1001.tar.bz2
https://repo.anaconda.com/pkgs/main/win-64/mkl-service-2.3.0-py37hb782905_0.conda
https://conda.anaconda.org/conda-forge/win-64/msgpack-python-1.0.0-py37heaa310e_1.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/numpy-1.18.4-py37hae9e721_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/pickleshare-0.7.5-py37hc8dfbb8_1001.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/pyqt-5.12.3-py37h1834ac0_3.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/pyrsistent-0.16.0-py37h8055547_0.tar.bz2
https://repo.anaconda.com/pkgs/main/win-64/pysocks-1.7.1-py37_0.conda
https://conda.anaconda.org/conda-forge/win-64/pywin32-ctypes-0.2.0-py37hc8dfbb8_1001.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/pyzmq-19.0.1-py37h453f00a_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/sniffio-1.1.0-py37hc8dfbb8_2.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/terminado-0.8.3-py37hc8dfbb8_1.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/bottleneck-1.3.2-py37hbc2f12b_1.tar.bz2
https://repo.anaconda.com/pkgs/main/win-64/cryptography-2.8-py37h7a1dbc1_0.conda
https://repo.anaconda.com/pkgs/main/win-64/pyqtgraph-0.10.0-py37h28b3542_3.conda
https://repo.anaconda.com/pkgs/main/win-64/scipy-1.4.1-py37h9439919_0.conda
https://repo.anaconda.com/pkgs/main/win-64/setuptools-45.2.0-py37_0.conda
https://repo.anaconda.com/pkgs/main/win-64/pyopenssl-19.1.0-py37_0.conda
https://repo.anaconda.com/pkgs/main/win-64/wheel-0.34.2-py37_0.conda
https://repo.anaconda.com/pkgs/main/win-64/pip-20.0.2-py37_1.conda
https://repo.anaconda.com/pkgs/main/win-64/urllib3-1.25.8-py37_0.conda
https://repo.anaconda.com/pkgs/main/win-64/requests-2.22.0-py37_1.conda
https://conda.anaconda.org/conda-forge/noarch/anyqt-0.0.10-py_1.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/attrs-19.3.0-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/backcall-0.1.0-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/cachecontrol-0.12.6-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/colorama-0.4.3-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/commonmark-0.9.1-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/cycler-0.10.0-py_2.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/decorator-4.4.2-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/defusedxml-0.6.0-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/dictdiffer-0.8.0-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/et_xmlfile-1.0.1-py_1001.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/h11-0.9.0-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/hpack-3.0.0-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/hstspreload-2020.5.19-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/hyperframe-5.2.0-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/h2-3.2.0-py37hc8dfbb8_1.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/httpcore-0.9.0-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/importlib_metadata-1.6.0-0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/jsonschema-3.2.0-py37hc8dfbb8_1.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/keyring-21.2.1-py37hc8dfbb8_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/ipython_genutils-0.2.0-py_1.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/traitlets-4.3.3-py37hc8dfbb8_1.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/jupyter_core-4.6.3-py37hc8dfbb8_1.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/jdcal-1.4.1-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/jinja2-2.11.2-pyh9f0ad1d_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/joblib-0.15.1-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/keyrings.alt-3.1-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/lockfile-0.12.2-py_1.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/nbformat-5.0.6-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/networkx-2.4-py_1.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/openpyxl-3.0.3-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/orange-canvas-core-0.1.12-py_1.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/pandocfilters-1.4.2-py_1.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/parso-0.7.0-pyh9f0ad1d_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/jedi-0.17.0-py37hc8dfbb8_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/prometheus_client-0.8.0-pyh9f0ad1d_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/prompt_toolkit-3.0.5-0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/pygments-2.6.1-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/pyparsing-2.4.7-pyh9f0ad1d_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/packaging-20.4-pyh9f0ad1d_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/python-dateutil-2.8.1-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/matplotlib-base-3.2.1-py37h911224e_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/baycomp-1.0.2-py_1.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/jupyter_client-6.1.3-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/orange-widget-base-4.5.0-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/python-louvain-0.13-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/pytz-2020.1-pyh9f0ad1d_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/pandas-1.0.3-py37h3bbf574_1.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/qtpy-1.9.0-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/rfc3986-1.3.2-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/httpx-0.13.1-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/send2trash-1.5.0-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/serverfiles-0.3.0-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/testpath-0.4.4-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/threadpoolctl-2.0.0-pyh5ca1d4c_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/scikit-learn-0.23.1-py37ha0ab79d_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/opentsne-0.4.3-py37h1fb7aa8_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/wcwidth-0.1.9-pyh9f0ad1d_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/prompt-toolkit-3.0.5-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/ipython-7.14.0-py37hc8dfbb8_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/ipykernel-5.3.0-py37h5ca1d4c_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/jupyter_console-6.1.0-py_1.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/qtconsole-4.7.4-pyh9f0ad1d_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/webencodings-0.5.1-py_1.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/bleach-3.1.5-pyh9f0ad1d_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/nbconvert-5.6.1-py37hc8dfbb8_1.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/notebook-6.0.3-py37hc8dfbb8_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/widgetsnbextension-3.5.1-py37_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/ipywidgets-7.5.1-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/jupyter-1.0.0-py_2.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/xlrd-1.2.0-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/xlsxwriter-1.2.8-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/orange3-3.25.1-py37h1834ac0_0.tar.bz2
https://conda.anaconda.org/conda-forge/noarch/zipp-3.1.0-py_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/importlib-metadata-1.6.0-py37hc8dfbb8_0.tar.bz2
https://conda.anaconda.org/conda-forge/win-64/conda-4.8.3-py37hc8dfbb8_1.tar.bz2
https://repo.anaconda.com/pkgs/main/noarch/tqdm-4.42.1-py_0.conda
https://repo.anaconda.com/pkgs/main/win-64/conda-package-handling-1.6.0-py37h62dcd97_0.conda

**Comment by ales-erjavec** [comment]
It seems this was already fixed recently on master (2a6651c1547127d001ccd93a003758c995b0f8e1) but is not yet released.
